EarthAsDMAnalyzer/plugins/EarthAsDMAnalyzer.cc

```diff
@@ -549,6 +582,10 @@ void EarthAsDMAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup&
             if (verbose_ > 4) LogPrint(MOD) << "        >> Wheel /  Station / Sector / Layer " << dtChamberId.wheel()
               << " / " << dtChamberId.station() << " / "  << dtChamberId.sector() << " / " << dtLayerId.layer();
             
+            
+            muon_dtSeg_Station_.push_back(dtChamberId.station());
+            muon_dtSeg_Sector_.push_back(dtChamberId.sector());
```

Comment:
this is under some `if` statement, right? what if those conditions are not satisfied? there should be some default to load for that case

Reply:
Oh maybe not, because this does not go into the ntuple